flow/flow.go

```diff
@@ -610,7 +617,7 @@ func (f *Flow) newTransportLayer(packet *gopacket.Packet) error {
 		f.Transport.A = strconv.Itoa(int(transportPacket.SrcPort))
 		f.Transport.B = strconv.Itoa(int(transportPacket.DstPort))
 		if config.GetConfig().GetBool("agent.capture_syn") {
-			f.TCPFlowMetric = &TCPMetric{}
+			f.TCPIPMetric = &TCPIPMetric{}
 			return f.updateTCPMetrics(packet)
```

Comment:
I think here we should check if BPF filter is on before calculating fragments, because this code is valid onkly when we do collect all the packets.
What do you think?

Reply:
checking BPF will not tell us that we are going to get or not all the packets, for ex: filtering only port 80 packets.